README.md

```diff
@@ -21,35 +22,41 @@ produces to complete your evaluation. For example, while HarbourBridge preserves
 primary keys, it does not currently translate other indexes. So, you'll need to
 add [Spanner secondary
 indexes](https://cloud.google.com/spanner/docs/secondary-indexes) if your SQL
-queries rely on PostgreSQL indexes that have been dropped. HarbourBridge is not
+queries rely on PostgreSQL/MySQL indexes that have been dropped. HarbourBridge is not
 intended for production database migration.
 
 To use the tool on a PostgreSQL database called mydb, run
 
 ```sh
-pg_dump mydb | harbourbridge
+pg_dump mydb | harbourbridge -driver=pgdump
```

Comment:
This is not needed because by default the driver is `pgdump`.

Reply:
We can say: 

```sh
# By default, the driver is "pgdump".
pg_dump mydb | harbourbridge
# Or, 
pg_dump mydb | harbourbridge -driver=pgdump
```

Reply:
What I recall is that we once had a discussion and default `pgdump` was removed. Currently there is no default condition and hence specifying a driver is mandatory. We can have couple of options:

-  We can add a error condition if driver is NULL and suggest user to pick up a driver:

```
func main() {
...

if driverName == "" {
fmt.Printf("\nNo driver (source DB) selected. Please specify a driver using '--driver' flag\n")
panic(fmt.Errorf("No driver (source DB) selected"))
}
fmt.Println("Using driver (source DB):", driverName)

...
}
```

- We can again add default condition for `pgdump` :
```
func main() {
...

if driverName == "" {
  driverName=PGDUMP
}
fmt.Println("Using driver (source DB):", driverName)

...
}
```

What I feel is that instead of having default condition (`pgdump` driver) and then error thrown by the code if dump file is not `pgdump` or the selected approach is `postgres` information schema, it's better to mandate the driver(which is currently done) and put a error condition for not specifying a driver.
What are your views on this? @hengfengli @nevinheintze

Reply:
Done. Updated as per the suggestion.